observability/monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -181,10 +181,7 @@ You can ensure that the containers that run monitoring components have enough CP
 
 You can configure these limits and requests for core platform monitoring components in the `openshift-monitoring` namespace and for the components that monitor user-defined projects in the `openshift-user-workload-monitoring` namespace.
 
-// The following module should only include core platform monitoring (CPM tags)
-include::modules/monitoring-about-specifying-limits-and-requests-for-monitoring-components.adoc[leveloffset=+2,tags=**;CPM;!UWM]
-// The following module should only include monitoring for user-defined projects (UWM tags)
-include::modules/monitoring-about-specifying-limits-and-requests-for-monitoring-components.adoc[leveloffset=+2,tags=**;!CPM;UWM]
+include::modules/monitoring-about-specifying-limits-and-requests-for-monitoring-components.adoc[leveloffset=+2]
```

Comment:
I'm unable to verify this link because this (origin) assembly is currently commented out in `_topic_map.yml`.